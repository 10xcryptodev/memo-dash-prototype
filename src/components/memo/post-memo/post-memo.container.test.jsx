import React from 'react'
import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'

import PostMemoContainer from './post-memo.container'
import { Form } from 'semantic-ui-react'

describe('<PostMemoContainer />', () => {
  let store
  beforeEach(() => {
    const mockStore = configureStore()
    store = mockStore({})
  })

  describe('Shallow rendering', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<PostMemoContainer />, { context: { store } })
      expect(wrapper).toMatchSnapshot()
    })
  })
})
