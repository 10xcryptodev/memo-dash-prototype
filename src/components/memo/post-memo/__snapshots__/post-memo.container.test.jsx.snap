// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostMemoContainer /> Shallow rendering renders without crashing 1`] = `
<WithFormik(PostMemoComponent)
  onMemoSubmitted={[Function]}
/>
`;
